projects/plugins/jetpack/modules/likes/queuehandler.js

```diff
@@ -189,16 +184,6 @@ function JetpackLikesMessageListener( event ) {
 			break;
 		}
 
-		case 'clickPostLike':
```

Comment:
This represents a further cleanup opportunity on the wpcom side:

fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Swf%2Syvxrf%2Qerfg%2Qabwdhrel.wf%3Se%3Q993p9pso%23817-og

This applies to some of the other custom message events here.